test_scripts/API/VehicleData/GetVehicleData/001_Success_flow.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is NOT allowed by Policies
+--
+-- Requirement summary:
+-- [GetVehicleData] As a mobile app wants to send a request to get the details of the vehicle data
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid GetVehicleData to SDL and this request is allowed by Policies
+-- SDL must:
+-- Transfer this request to HMI and after successful response from hmi
+-- Respond SUCCESS, success:true to mobile application
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Local Variables ]]
+
+local rpc = {
+  name = "GetVehicleData",
+  params = {
+    engineOilLife = true
+  }
+}
+
+local function processRPCSuccess(self)
+  local mobileSession = common.getMobileSession(self, 1)
+  local cid = mobileSession:SendRPC(rpc.name, rpc.params)
+  EXPECT_HMICALL("VehicleInfo." .. rpc.name, rpc.params)
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", { engineOilLife = 52.3 })
+    end)
+  mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
```

Comment:
@AByzhynar probably we should also check `engineOilLife `parameter to verify that SDL correctly forawrds it from HMI to Mobile

Reply:
@AByzhynar please also check that for other test scripts in this pull request

Reply:
@AKalinich-Luxoft Right. Will add it. Thanks.

Reply:
@AKalinich-Luxoft It is checked in other scripts

Reply:
@AKalinich-Luxoft Done in 891857a555e71a421a01111967d75e5bf7fbc92b